noir-projects/aztec-nr/aztec/src/oracle/nullifier_key.nr

```diff
@@ -26,3 +27,26 @@ pub fn get_nullifier_keys(account: AztecAddress) -> NullifierKeys {
 pub fn get_app_nullifier_secret_key(account: AztecAddress) -> Field {
     get_nullifier_keys_internal(account).app_nullifier_secret_key
 }
+
+// TODO (#6176): Replace get_nullifier_keys above with this once we no longer get nullifier keys with address
+#[oracle(getNullifierKeysWithNpkMH)]
+fn get_nullifier_keys_with_npk_m_h_oracle(_master_nullifier_public_key_hash: Field) -> [Field; 3] {}
+
+unconstrained fn get_nullifier_keys_with_npk_m_h_internal(master_nullifier_public_key_hash: Field) -> NullifierKeys {
+    let result = get_nullifier_keys_with_npk_m_h_oracle(master_nullifier_public_key_hash);
+    NullifierKeys {
+        account: AztecAddress::zero(),
```

Comment:
This will also be fixed with #6176 yes? BTW you have marked that issue as a duplicate of another issue, but it is still easy to find the proper issue so think it is ok if there are no other changes needed to keep it that way.